index.html

```diff
@@ -129,37 +141,114 @@ <h1 class="text-3xl font-bold text-gray-800 mb-2 text-center">🛡️ Scam & Phi
-    <p>This service is provided for free by <a href="https://bettergov.ph" target="_blank"
+    <p data-i18n="footer">This service is provided for free by <a href="https://bettergov.ph" target="_blank"
         rel="noopener noreferrer">BetterGov.ph</a></p>
   </div>
 
   <script>
+    // Translations
+    const translations = {
+      en: {
+        title: "🛡️ Scam & Phishing Detector",
+        subtitle: "Test your messages for spam and phishing attempts",
+        placeholder: "Paste your email or SMS text here to check if it's spam or phishing...\n\nExample: 'URGENT: Your account has been suspended. Click here to verify your identity immediately!'",
+        analyzeButton: "Analyze Text",
+        examplesTitle: "Try these examples:",
+        example1: "GCash Scam",
+        example2: "Delivery SMS",
+        example3: "Bank Alert",
+        example4: "Crypto Scam",
+        example5: "Prize Scam",
+        example6: "Job Scam",
+        example7: "Legitimate",
+        footer: "This service is provided for free by BetterGov.ph",
+        analyzing: "Analyzing...",
+        confidence: "Confidence",
+        enterText: "Please enter some text to analyze",
+        completeCaptcha: "Please complete the CAPTCHA verification"
+      },
+      tl: {
+        title: "🛡️ Detektor ng Scam at Phishing",
+        subtitle: "Suriin ang inyong mensahe para sa spam at phishing",
+        placeholder: "I-paste dito ang iyong email o SMS upang suriin kung ito ay spam o phishing...\n\nHalimbawa: 'MADALIANG PABATID: Ang iyong account ay nasuspinde. I-click dito upang agad na patunayan ang iyong pagkakakilanlan!'",
+        analyzeButton: "Suriin ang Teksto",
+        examplesTitle: "Subukan ang mga ito:",
+        example1: "GCash Scam",
+        example2: "Delivery SMS",
+        example3: "Bank Alert",
+        example4: "Crypto Scam",
+        example5: "Prize Scam",
+        example6: "Job Scam",
+        example7: "Lehitimo",
+        footer: "Ang serbisyong ito ay libre mula sa BetterGov.ph",
+        analyzing: "Sinusuri...",
+        confidence: "Katiyakan",
+        enterText: "Mangyaring maglagay ng teksto",
+        completeCaptcha: "Mangyaring kumpletuhin ang CAPTCHA verification"
+      }
+    };
+
+    // Language state
+    let currentLang = localStorage.getItem('preferredLanguage') || 'en';
+
+    // Language functions
+    function changeLanguage(lang) {
+      currentLang = lang;
+      localStorage.setItem('preferredLanguage', lang);
+      updateUILanguage();
+    }
+
+    function updateUILanguage() {
+      const t = translations[currentLang] || translations.en;
+      
+      // Update all elements with data-i18n attribute
+      document.querySelectorAll('[data-i18n]').forEach(elem => {
+        const key = elem.getAttribute('data-i18n');
+        if (t[key]) {
+          if (key === 'footer') {
+            elem.innerHTML = t[key];
+          } else {
+            elem.textContent = t[key];
+          }
+        }
```

Comment:
nit: this can be refactored with the _guard clause_ to reduce indentation:

```suggestion
        if (!t[key]) {
          return;
        }

        if (key === 'footer') {
          elem.innerHTML = t[key];
        } else {
          elem.textContent = t[key];
        }
```